internal/cache/lru_cache.go

```diff
@@ -50,10 +54,14 @@ func NewLRUCache(capacity int) *LRUCache {
 //
 // The element is removed from the map too because
 // it might have stale node values.
-func (lru *LRUCache) GetElement(element interface{}) error {
+//
+// Error is returned only if the element doesn't exist in the cache.
```

Comment:
Maybe we shouldnt be so blunt about it being the `owner`, it can technically be anything in the future. Ill mention, it returns associated data will be returned along with an error.